app/src/main/java/uk/ac/bath/petmatch/Utils/UserCapabilitiesFragment.java

```diff
@@ -1,17 +1,45 @@
 package uk.ac.bath.petmatch.Utils;
```

Comment:
Please remove fragment from Utils package, either to the main package or create Fragments package